Common/Platform/x86x64/Linux/PlatformDefines.h

```diff
@@ -37,6 +37,50 @@ using tchar = char;
 using tstring = std::string;
 #endif
 
+#ifdef __linux__
+#define WINAPIV
+#elif _WIN32
+#define WINAPIV __cdecl
```

Comment:
no macro LINUX or _LINUX. It is linux
http://stackoverflow.com/questions/6649936/c-compiling-on-windows-and-linux-ifdef-switch
or
http://sourceforge.net/p/predef/wiki/OperatingSystems/

I tested